test/freenet/clients/http/utils/PebbleUtilsTest.java

```diff
@@ -15,7 +15,7 @@ public class PebbleUtilsTest {
 	@Test
 	public void addChildAddsCorrectlyEvaluatedSimpleTemplateToHtmlNode() throws IOException {
 		PebbleUtils.addChild(emptyParentNode, "pebble-utils-test-simple", model, null);
-		assertThat(emptyParentNode.generate(), equalTo("Test!\n"));
+		assertThat(emptyParentNode.generate(), equalTo("Test!" + System.lineSeparator()));
```

Comment:
Now, this is actually interesting. What we do want here is to test that the template substitution works correctly, which means that we very much do want to avoid using system-dependent behaviour here; however, that also means that we need to make sure that the source of the template contains exactly the line endings that we expect in the test. I guess the real fix here is to make sure that Git doesn’t change the line endings of the file.

Reply:
I hadn’t thought of git changing the line endings in the test files! :facepalm: 

That won’t happen in an unpacked tarball, so we’d get test failures there.

@alvonellos can you check whether the linesep can also be fixed by adding a `.gitattributes` file for our expected test output similar to this? https://shzhangji.com/blog/2022/08/31/configure-git-line-endings-across-oses/

Reply:
I'm looking into this.